src/app/core/footer/footer.component.scss

```diff
@@ -130,8 +130,14 @@
     font-size: $font-size-regular;
     color: $show-more-color;
     cursor: pointer;
+    text-decoration: none;
```

Comment:
I wonder if this justifies a new mixin to be included here (and anywhere links should not be underlined until hover).  Does not block merge (approving this PR).

Reply:
Good idea, I added a mixin.